libcni/api.go

```diff
@@ -196,6 +209,7 @@ type cachedInfo struct {
 	Config         []byte                 `json:"config"`
 	IfName         string                 `json:"ifName"`
 	NetworkName    string                 `json:"networkName"`
+	NetNS          string                 `json:"netns,omitempty"`
```

Comment:
This PR introduces `NetNS` in cachedInfo as new field. I'm a bit curious that this code works with previous version CNI cache, that does not have `NetNS` field. Could you please double check that?

Reply:
Good catch. I've created a small program to validate the json marshaling for this change [here](https://go.dev/play/p/z71ZJfB29a_1). It seems fine.